webrtc.html

```diff
@@ -2354,11 +2356,124 @@
 
           <dd>This indicates the index (starting at zero) of the m-line in the
           SDP this candidate is associated with.</dd>
+
+          <dt>DOMString foundation</dt>
+
+          <dd>A unique identifier that allows ICE to correlate candidates that
+          appear on multiple <code><a>RTCIceTransport</a></code>s.</dd>
+
+          <dt>unsigned long priority</dt>
+
+          <dd>The assigned priority of the candidate. This is automatically
+          populated by the browser.</dd>
```

Comment:
If `addIceCandidate()` ignores these values, then they don't need to be added here.  They can be added as attributes on a new dictionary that inherits from `RTCIceCandidate`, a new type that would be an argument to onicecandidate events only.  That avoids the implication that these parameters are required, or that they might need to be consistent with the main `candidate` string.